test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/ProjectDefination.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Drawing.Text;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using EnvDTE;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.Services.Profile;
+
+
+namespace NuGet.Tests.Apex.NuGetEndToEndTests
+{
+    internal class WebSiteProjectItems
+    {
+        public List<WebSiteProjectItem> InitProjectItems()
+        {
+            List<WebSiteProjectItem> _websiteProjectItems = new List<WebSiteProjectItem>();
+            _websiteProjectItems.Add(new WebSiteProjectItem() { Template=ProjectTemplate.WebSiteEmpty, ProjectName="WebSiteEmpty", TargetFramework=ProjectTargetFramework.V48, PackageName="log4net", PackageVersion="2.0.13" });
+            _websiteProjectItems.Add(new WebSiteProjectItem() { Template=ProjectTemplate.WebSite, ProjectName="WebSite", TargetFramework=ProjectTargetFramework.V48, PackageName="Log4Net.Async", PackageVersion="2.0.3" });
+            _websiteProjectItems.Add(new WebSiteProjectItem() { Template=ProjectTemplate.WebSiteWCF, ProjectName="WebSiteWCF", TargetFramework=ProjectTargetFramework.V48, PackageName="log4net.Ext.Json", PackageVersion="2.0.9.1" });
+            _websiteProjectItems.Add(new WebSiteProjectItem() { Template=ProjectTemplate.WebSiteDynamicDataEntityFramework, ProjectName="WebSiteEntities", TargetFramework=ProjectTargetFramework.V48, PackageName="log4net", PackageVersion="2.0.13" });
+            _websiteProjectItems.Add(new WebSiteProjectItem() { Template=ProjectTemplate.WebSiteDynamicDataLinqToSql, ProjectName="WebSiteLinq", TargetFramework=ProjectTargetFramework.V45, PackageName="Log4Net.Async", PackageVersion="2.0.3" });
```

Comment:
Any particular reason why we're testing net45? I see the rest of the templates are net48, so I was wondering if there's any particular reason for that.

Reply:
Hi @nkolev92, for the last project-**ASP.NET Dynamic Data Linq to SQL Web Site**, it only support .NET Framework 4.5.2 and the framework cannot be changed when creating the project manually as below.
![image](https://user-images.githubusercontent.com/20632625/190295930-2acaab55-009c-407b-9744-6c68ffc8c67f.png)

Reply:
Thanks for the explanation. Makes sense.  Might be a good idea to add a small comment in the test case declaration.